opensearch-observability/src/main/resources/observability-mapping.yml

```diff
@@ -54,6 +54,14 @@ properties:
         fields:
           keyword:
             type: keyword
+  application:
+    type: object
+    properties:
+      name:
+        type: text
+        fields:
+          keyword:
+            type: keyword
```

Comment:
might also need to add logic to update mapping of indices created before this change

Reply:
Oh yea I forgot about that! Will add that!